src/Microsoft.Identity.Web/AccountExtensions.cs

```diff
@@ -17,7 +17,7 @@ public static class AccountExtensions
         /// </summary>
         /// <param name="account">The IAccount instance.</param>
         /// <returns>A <see cref="ClaimsPrincipal"/> built from IAccount.</returns>
-        public static ClaimsPrincipal ToClaimsPrincipal(this IAccount account)
+        public static ClaimsPrincipal? ToClaimsPrincipal(this IAccount? account)
```

Comment:
Public API changes are backwards compatible.

Reply:
Here and in similar methods, we can also use [attributes](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/attributes/nullable-analysis) like `[return: NotNullIfNotNull("account")]